tests/output_filters/opendns/related_domains_test.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+import testify as T
+from osxcollector.output_filters.opendns. \
+    related_domains import RelatedDomainsFilter
+from tests.output_filters.run_filter_test import RunFilterTest
+
+
+class RelatedDomainsFilterTest(RunFilterTest):
+
+    def test_no_domains(self):
+        input_blobs = [
+            {'fungo': 'dingo', 'bingo': [11, 37], 'banana': {'a': 11}},
```

Comment:
Can you confirm that you only want one dict in the input_blob here? 
For all other tests, you include this 2nd dict: {'span': 'div', 'head': ['tail', 22], 'orange': {'lemmon': 'zits'}}

Also, this really doesn't matter, but "lemmon" is not a word (although Lemmon is a proper name, apparently).